### PR TITLE
changed names of files and input code for group 11

### PR DESCRIPTION
**Pull request by griffindavis55**

### CODE REVIEW

**Review by andrewlinington — APPROVED** [review]
This is ready to merge just awaiting a secondary review

File: main.c [hunk 1]
```diff
@@ -302,10 +303,7 @@ void runAllTests() {
 
     runXPlaneReflectionTests(); // X plane reflection tests
 
-    runYPlaneReflectionTests(); // Group 11 tests
```
Comment (andrewlinington) [inline main.c]: whoops that needs added back in my bad

**Review by andrewlinington — CHANGES_REQUESTED** [review]
tests seem to actually fail based on how your tests were run as of now  (not a valgrind thing but your actual tests)

**Review by aidkvndfj — APPROVED** [review]

**Review by cc0407 — APPROVED** [review]
Lookin' Good

**Review by andrewlinington — APPROVED** [review]
Perfect its gonna be merged now

**Pull request merged**